src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: New Relic Kubernetes components
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Kubernetes components
+metaDescription: "Learn what components are deployed after installing Kubernetes."
+---
+
+import kubernetesv3Integration from 'images/kubernetes_diagram_v3-integration.png'
+
+
+After deploying Kubernetes you have a cluster, which consists of a set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node. Kubernetes runs your workload by placing containers into pods to run on nodes.
+
+The control plane is the container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers. The control plane manages also the worker nodes and the pods in the cluster.
+
+There're four distinct components that need to be monitored in a Kubernetes environment each with their specificities and challenges:
+
+* Infrastructure
+* Containers
+* Applications
+* Kubernetes clusters
+
+The Kubernetes integration of New Relic gives you full observability into the health and performance of your environment, no matter whether you run Kubernetes on-premises or in the cloud. It gives you visibility about Kubernetes namespaces, deployments, replica sets, nodes, pods, and containers. Metrics are collected from different sources:
+
+* `kube-state-metrics service` provides information about state of Kubernetes objects like namespace, replicaset, deployments, and pods (when they are not in running state).
+* `/stats/summary` kubelet endpoint gives information about network, errors, memory and CPU usage.
+* `/pods` kubelet endpoint provides information about state of running pods and containers.
+* `/metrics/cadvisor` cAdvisor endpoint provides missing data that is not included in the previous sources.
+* `/metrics` from control plane components: ETCD, controllerManager, apiServer, and scheduler.
+
+The main component of the integration is the `newrelic-infrastructure` DaemonSet. This component is divided in these other components:
+* `nrk8s-ksm`
+* `nrk8s-kubelet`
+* `nrk8s-controlplane`
+
+The first one is a deployment and the others two are DaemonSets.Each of the components has these containers:
+
+1. A container for the integration, responsible for collecting metrics.
+2. A container with the New Relic infrastructure agent, which is used to send the metrics to New Relic.
+
+<img
+  title="Diagram showing the integration installed in a 3 nodes cluster"
+  alt="Diagram showing the integration installed in a 3 nodes cluster"
+  src={kubernetesv3Integration}
+/>
```

Comment:
I suggest to get rid of this diagram. It does not fit particularly well with the visual style of the docs, and uses internal terminology that can be very confusing to users. For example, here `kubelet` does not refer to the `kubelet`, but to our kubelet scraper. If we want to show a diagram here we should make one that is externally-consumable.

Reply:
Removed it.